src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -325,8 +325,8 @@ const buildWhereClause = ({ schema, query, index }) => {
       const point = fieldValue.$nearSphere;
       const distance = fieldValue.$maxDistance;
       const distanceInKM = distance * 6371 * 1000;
-      patterns.push(`ST_distance_sphere($${index}:name::geometry, POINT($${index + 1}, $${index + 2})::geometry) <= $${index + 3}`);
-      sorts.push(`ST_distance_sphere($${index}:name::geometry, POINT($${index + 1}, $${index + 2})::geometry) ASC`)
+      patterns.push(`ST_distance_sphere($${index}:name::geometry, POINT($${index + 2}, $${index + 1})::geometry) <= $${index + 3}`);
+      sorts.push(`ST_distance_sphere($${index}:name::geometry, POINT($${index + 2}, $${index + 1})::geometry) ASC`)
       values.push(fieldName, point.longitude, point.latitude, distanceInKM);
       index += 4;
```

Comment:
This looks fine. 
Although I think it might be more readable to change the order of latitude/longitude when pushing them in `values`. 

So, leaving lines 328/329 as they originally were and replacing `values.push(fieldName, point.longitude, point.latitude, distanceInKM);` with `values.push(fieldName, point.latitude, point.longitude, distanceInKM);` on line 330